packages/ui/src/controllers/ui/ui-desktop.controller.tsx

```diff
@@ -76,6 +76,8 @@ export class DesktopUIController extends Disposable implements IDesktopUIControl
                 this._renderManagerService.currentRender$.subscribe((renderId) => {
                     if (renderId) {
                         const render = this._renderManagerService.getRenderById(renderId)!;
+                        if (!render.unitId) return;
```

Comment:
Under what circumstances does unitId not exist?

Reply:
Cell editors.